content/docs/command-reference/plot/diff.md

```diff
@@ -0,0 +1,130 @@
+# plot diff
+
+Show multiple versions of
+[continuous metrics](/doc/command-reference/plot#continous-metrics) by plotting
```

Comment:
I wonder whether we want to name them `continuous`. This word applies to functions. What about, for example, confusion matrix? Data for that type of plot is not continuous.

Reply:
Agreed with @pared . Probably even plain explicit "non-scalar metric" would be better?

Reply:
yeah. it will be changed. all the terminology around continuous will be removed in the next iteration.

Reply:
- [ ] So let's leave "continuous" in this PR so we have a plot cmd ref for now, and update it in a following PR.